manual/src/threelang/explorer/contracts.md

```diff
@@ -0,0 +1,32 @@
+# Contracts Action
+Query and filter contracts on the chain.
+
+## Filter contracts
+- action name: !!explor.contracts.filter
```

Comment:
I'm actually thinking that we should not use explorer here but rather tfchain. Because all it does is return tfchain information. I think we should integrate this into tfchain.

Reply:
> *explorer

yes. this was limited by `actionparser` in crystlalib to have at most 6 chars. i will edit it there

Reply:
I don't think there should be a maximum limit. Maybe you can remove that?

Reply:
@robvanmieghem can you also share your 2 cents on the explorer? What do you think? Should we hide it under tfchain?